server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/measures/MeasuresPage.java

```diff
@@ -17,83 +17,81 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarqube.pageobjects.measures;
+package org.sonarqube.qa.util.pageobjects.measures;
+
 import com.codeborne.selenide.CollectionCondition;
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Selenide;
 import com.codeborne.selenide.SelenideElement;
 import org.openqa.selenium.Keys;
 
-import static com.codeborne.selenide.Condition.exist;
-import static com.codeborne.selenide.Condition.text;
-import static com.codeborne.selenide.Selenide.$;
-import static com.codeborne.selenide.Selenide.$$;
-
 public class MeasuresPage {
   public MeasuresPage() {
-    $("#component-measures").should(exist);
+    Selenide.$("#component-measures").should(Condition.exist);
   }
 
   public MeasuresPage displayBubbleChart(String title) {
-    SelenideElement bubblechart = $("#component-measures .measure-overview-bubble-chart");
-    bubblechart.$(".measure-overview-bubble-chart-title").shouldHave(text(title));
+    SelenideElement bubblechart = Selenide.$("#component-measures .measure-overview-bubble-chart");
+    bubblechart.$(".measure-overview-bubble-chart-title").shouldHave(Condition.text(title));
     return this;
   }
 
   public MeasuresPage measureHasValue(String measure, Integer value) {
-    SelenideElement sidebar = this.getSideBar();
-    sidebar.$("#measure-" + measure + "-name").should(exist);
-    sidebar.$("#measure-" + measure + "-value").should(exist).shouldHave(text(value.toString()));
+    SelenideElement sidebar = getSideBar();
+    sidebar.$("#measure-" + measure + "-name").should(Condition.exist);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "-name" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "#measure-" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)